optuna/pruners/hyperband.py

```diff
@@ -50,6 +50,10 @@ class HyperbandPruner(BasePruner):
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
     Args:
+        max_resource:
+            A parameter for specifying the maximum resource allocated to a trial noted as :math:`R`
+            in the paper. This value is exactly same with the maximum iteration steps (e.g.,
```

Comment:
A formulation suggestion. "is exactly" could be interpreted as something that this class can guarantee. We could formulate it more as a suggestion/hint to the user.
```suggestion
            in the paper. This value represents and should match the maximum iteration steps (e.g.,
```